h2o-core/src/main/java/hex/Model.java

```diff
@@ -209,7 +209,17 @@ public void setTrain(Key<Frame> train) {
       AUTO, Random, Modulo, Stratified
     }
     public enum CategoricalEncodingScheme {
-      AUTO(false), OneHotInternal(false), OneHotExplicit(false), Enum(false), Binary(false), Eigen(false), LabelEncoder(false), SortByResponse(true), EnumLimited(false);
```

Comment:
Can you please confirm this was just reformatted or is there also another change?

Reply:
just reformatting